### ISSUE TITLE
Pact-V5 doesn't allow untyped schemas

### ISSUE THREAD

**Issue by CryptoPascal31** [kadena-io/pact-5#80]
## Issue description

In Pact legacy, untyped schemas were allowed:
Example:
```pact
(module A GOV 
  (defcap GOV() true) 
     
  (defschema my-schema
      a
      b)
)
```
Pact-v5 refuses it. I'm not sure, it was a good practice... But it breaks compatibility.

**Comment by jmcardon** [comment]
So we really wanted to do away with them, but it does seem like we can't.

Good catch, albeit I already implemented them in https://github.com/kadena-io/pact-5/pull/48 and they just did not make it in for alpha release.

I will cherry pick the changes from that PR and open a new one.

**Comment by jmcardon** [comment]
Fixed in #48.
